.github/ISSUE_TEMPLATE/release.md

```diff
@@ -10,44 +10,28 @@ Please check items along as you follow the release process.
 
 #### Overview
 
-This is a checklist for releases. This is filled in by both the releaser and the reviewer where necessary.
+This is a checklist for releases.
 
-#### Preparation
+#### Release candidate
 
 - [ ] Create a [documentation release issue](https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/new?title=Release+v3.x.x&labels=release&template=release.md).
-- [ ] Run `tools/bin/prepare_release minor` or `tools/bin/prepare_release patch`, depending on the kind of release.
-- [ ] Follow the instructions.
-
-#### Check 1 (for reviewers)
-
-- [ ] The Changelog is complete i.e., contains only the changes that are in the release (not more/less).
-- [ ] The version files are correctly updated.
+- [ ] Run the `create-release-branch.yml` Github Action.
```

Comment:
Probably put the name here directly, as the file name is not visible in the UI. Same below.